testmodels/src/main/java/com/amplifyframework/testmodels/cpk/AmplifyModelProvider.java

```diff
@@ -0,0 +1,53 @@
+package com.amplifyframework.testmodels.cpk;
```

Comment:
Missing copyright.

Reply:
Hmm, I think that would be ok here. These are codegen and we don't add licenses to our codegen classes that typically go into customer classes. I don't necessarily want to touch codegen'ed files.

Reply:
On second thought, looks like we've added them on all the others so went ahead and added to all the new codegen files.